index.js

```diff
@@ -78,6 +89,9 @@ function createError (data, response, callback) {
   var error = httpError(response.statusCode);
   if (!data) return callback(error);
   if (data) {
+    if (Array.isArray(data)) {
+      data = data[0]
+    }
```

Comment:
Is this desired? Wouldn't this break returning an array of data with `json: true`?